recipes/scg/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "scg" %}
+{% set version = "0.3.1" %}
+
+package:
+    name: {{ name|lower }}
+    version: {{ version }}
+
+source:
+    url: https://bitbucket.org/aroth85/scg/downloads/{{ name }}-{{ version }}.tar.gz
+    md5: 6f0fb91266de66d6a9d73fb741f49e13
+
+build:
+    script: $PYTHON setup.py install
+    number: 0
+    skip: True # [not py27]
+#    noarch: python # tried building with it, causes TESTS to fail, no clue why
+
+requirements:
+    build:
+        - python
+        - pandas >=0.16
+        - numpy >=1.9.2
+        - scipy >=0.15
+        - pyyaml
+
+    run:
+        - python
+        - pandas >=0.16
+        - numpy >=1.9.2
+        - scipy >=0.15
+        - pyyaml
+        - libgcc
```

Comment:
is libgcc really needed? You are don't have gcc as build-time dependency. Are you compiling anything?